packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -93,6 +93,11 @@ export class BlobManager {
         const storageId = this.redirectTable?.get(blobId) ?? blobId;
         assert(this.hasBlob(storageId), 0x11f /* "requesting unknown blobs" */);
 
+        // logging readBlob error if there is any
+        const readBlobP = this.getStorage().readBlob(storageId);
+        readBlobP.catch(
+            (error) => this.logger.sendErrorEvent({ eventName:"storageService_readBlobError" }, error));
```

Comment:
This seems wrong. Why do we read blob here? This extra call is unnecessary, and logging added does not represent what actually happens when handle is resolve, as it might succeed here, and fail when BlobHandle is resolved below.
Any telemetry should be added where we actually load blobs today.

Reply:
That said, consider adding storageId property as "id" to event - that way we can see if it's same blob or not across many sessions.